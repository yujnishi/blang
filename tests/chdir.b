main() {
    if ( !fork() ) execl("/usr/bin/pwd","pwd",0);
    wait();
    chdir("/");
    if ( !fork() ) execl("/usr/bin/pwd","pwd",0);
    wait();
}
