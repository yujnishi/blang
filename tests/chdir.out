/home/ec2-user/work/blang/tests
/
